lib/arc/actions/validate.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Arc.Actions.Validate do
+
+  defmacro __using__(_) do
+    quote do
+      def valid?(args), do: Arc.Actions.Validate.valid?(__MODULE__, args)
+    end
+  end
+
+  def valid?(definition, {file, scope}) when is_binary(file) or is_map(file) do
+    case Arc.File.new(file) do
```

Comment:
I had not thought about that. Thanks for reviewing. I will think about a solution to this problem.